frontend/src/components/samples/SampleQCFlag.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import {Popover, Button} from "antd";
+import {CheckOutlined, CloseOutlined} from "@ant-design/icons";
+
+const getFlagContent = (flags) => {
+  return  (
+    <div>
+      <p> Quantity Flag: {flags.quantity ? "Passed" : "Failed"}</p>
```

Comment:
Use the keys and values instead of harcoding the names of the flags here in order to keep this generic

Reply:
the values are booleans, not strings